demos/demo-minimal-js/index.html

```diff
@@ -89,6 +114,27 @@
         border-top: 1px solid rgba(0, 0, 0, 0.1);
         padding-top: 0.5rem;
       }
+      span.beta-badge {
```

Comment:
Would it be possible to keep the size of the incoming call buttons consistent with the existing buttons so that we don't need to scroll within the demo widget? <img width="393" alt="image" src="https://github.com/HubSpot/calling-extensions-sdk/assets/23175119/32e8e138-a7aa-4a46-ba1a-a4a39e71846e">

Reply:
i think the instruction text above is making it scrolling.

Reply:
Even after reducing instruction text, its bit scrolling.

Reply:
done